src/components/Contacts/ContactDetails/ContactDetailsTab/ContactDetailsTab.stories.tsx

```diff
@@ -17,9 +19,15 @@ const contactId = '222';
 
 export const Default = (): ReactElement => {
   return (
-    <GqlMockedProvider<ContactDetailsTabQuery>>
-      <ContactDetailsTab accountListId={accountListId} contactId={contactId} />
-    </GqlMockedProvider>
+    <MuiThemeProvider theme={theme}>
```

Comment:
I haven't done this yet, but is this needed to provide the theme to the child components in storybook? What happens if you don't provide this, does it just use the default theme values?

Reply:
@TheNoodleMoose yes if you don't use it you will not get the theme that we create.

Reply:
just a follow up to this, this is already defined in `.storybook/preview.js`, there shouldn't be a need to specify it for each story.